charts/drax/charts/golang-nkafka/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- include
+      "accelleran.common.deployment"
+      (fromYaml (include "accelleran.golang-nkafka.deployment.args" $))
+-}}
+
+{{- define "accelleran.golang-nkafka.deployment.args" -}}
+{{- $ := . -}}
+{{- $fullname := include "accelleran.common.fullname" (dict "top" $) -}}
+
+top:
+  {{ $ | toYaml | nindent 2 }}
+
+initContainers:
+- {{ fromYaml (include "accelleran.common.init.kafka" (dict "top" $ "values" (fromYaml (include "accelleran.golang-nkafka.disabledLicenseValues" .)))) | toYaml | nindent 2 }}
+- {{ fromYaml (include "accelleran.common.init.nats" (dict "top" $ "values" (fromYaml (include "accelleran.golang-nkafka.disabledLicenseValues" .)))) | toYaml | nindent 2 }}
+- {{ fromYaml (include "accelleran.golang-nkafka.init.config" $) | toYaml | nindent 2 }}
+
+env:
+- name: "NATS_KAFKA_BRIDGE_CONFIG"
+  value: "/nkafka.conf"
+
+volumes:
+- name: {{ $fullname }}-config-volume
+  emptyDir: {}
+- name: {{ $fullname }}-config
+  configMap:
+    name: {{ $fullname }}-config
+{{ if .Values.config.natsKafkaTopics.override }}
+- name: {{ $fullname }}-kafkatopics
+  configMap:
+    name: {{ $fullname }}-kafkatopics
+{{ end }}
+volumeMounts:
+- name: {{ $fullname }}-config
+  mountPath: /nkafka.conf
+  subPath: "nkafka.conf"
+- name: {{ $fullname }}-config-volume
+  mountPath: /config
+{{- end -}}
+
+
+{{- define "accelleran.golang-nkafka.disabledLicenseValues" -}}
+{{-
+  (mergeOverwrite
+    $.Values
+    (dict
+      "accelleranLicense"
+      (dict
+        "enabled" false
+      )
+    )
+  ) | toYaml
+-}}
+{{- end -}}
```

Comment:
Just trying to work out what this is for.. Is this needed so that initContainers of licensed software Pods don't try to use `-licensed` images for the initImage?

Reply:
Exactly